src/read_input.f90

```diff
@@ -580,7 +580,7 @@ end subroutine read_input_data
  subroutine consistency_check_number_of_particles_comp
  if(all(ppc>=1)) then
   call from_ppc_to_npx_npy_npz
- elseif(all(np_per_zc)==-1) then
+ elseif(all(np_per_zc==-1)) then
```

Comment:
again, we are already with this fix in master. Do you have any idea why it is highlighted?